src/safeds/data/tabular/transformation/_discretizer.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from sklearn.preprocessing import KBinsDiscretizer as sk_KBinsDiscretizer
+
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation._table_transformer import TableTransformer
```

Comment:
```suggestion
from safeds.data.tabular.transformation import TableTransformer
```

Reply:
tests don't work if I import it like this and it's also importet with `._table_transformer` in the other transformers